source/extended-json.rst

```diff
@@ -515,99 +763,63 @@ forms are distinguishable from that of DBRef, so this specification creates a
 new format: ``{"$dbPointer": {"$ref": <namespace>, "$id": {"$oid": <hex
 string>}}}``.
 
-Finally, if a BSON type fell into category (3), this specification proposes a
-format for the type. The following new extended JSON forms are introduced by
-this spec:
+Newly-added type wrappers
```

Comment:
Oh, I think I see. mongoexport / mongoimport, and I guess some drivers, already support $numberLong. OK, you can ignore this then.